authentication/account/service/user.go

```diff
@@ -31,9 +31,9 @@ type userServiceImpl struct {
 	base.BaseService
 }
 
-// ResetDeprovisioned sets User.Deprovisioned to false
+// ResetBanned sets User.Banned to false
 func (s *userServiceImpl) ResetDeprovision(ctx context.Context, user repository.User) error {
```

Comment:
You changed the comment but the method is still ResetDeprovision.

Reply:
ah right! Let me fix that ;)

Reply:
addressed in f1fff0e